packages/bun-error/index.tsx

```diff
@@ -1137,7 +1088,7 @@ const OverlayMessageContainer = ({
 function copyToClipboard(input: string | Promise<string>) {
   if (!input) return;
 
-  if (input && typeof input === "object" && "then" in input) {
+  if (typeof input === "object" && "then" in input) {
```

Comment:
If `input` was null, this code would throw at runtime so the `input &&` is necessary

Reply:
The `if (!input) return` guard on line 1086 should cover that case, so the additional check shouldn't be necessary